packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -223,6 +223,19 @@ exports.mapTagsCategoriesToTaxonomies = entities =>
     return e
   })
 
+exports.mapPagesToParentPages = entities => {
+  const pages = entities.filter(e => e.__type === `wordpress__PAGE`)
```

Comment:
Should this be limited to pages only? Custom post types can support page attributes (and if you really want to you can enable this for posts for some reason) in wordpress.

Reply:
@pieh Do we agree that the mapping would be inside the post type only ? Like a page linking a page, but not a page linking a post.

Reply:
I think this is by design in wordpress, but yes - I surely do agree on that. I don't think we could handle this for other post types without this assumption, because wordpress ids in posts/pages/custom post types are unique, but taxonomy terms can have same ids as posts and we don't really know which entities are taxonomy terms and which are posts/pages/entries of custom post types.

Does it make sense?

I just thought that if we would just check if `wordpress_parent` is truthy and try to link to other entity of same type - this would also automatically enable this feature for categories and other hierarchical taxonomies, right?

Reply:
Done.